src/packages/@workspaces/features/environment-list/components/list-item/ListItem.svelte

```diff
@@ -160,7 +159,7 @@
   handleModalState={handleEnvironmentPopUpCancel}
 >
   <div class="text-lightGray mb-1 sparrow-fs-14">
-    <p>
+    <p style="font-size: 14px; font-weight: 400;">
       Are you sure you want to delete this Environment? <span
         style="font-weight:700;"
```

Comment:
dont use font size in pixel, we have a class `text-fs-14` that holds value in rem